src/main/java/net/minecraftforge/fml/common/FMLCommonHandler.java

```diff
@@ -689,6 +689,12 @@ public IThreadListener getWorldThread(INetHandler net)
         return sidedDelegate.getWorldThread(net);
     }
 
+    @Nullable
+    public IThreadListener getMinecraftThread()
```

Comment:
This should be `getClientThread`, in my opinion. The current name suggests that it is for the thread that Minecraft is running on - be that a client or server.

Reply:
What? Its clearly used to get the server thread as well. Maybe `getMainThreadListener`?